orchagent/vnetorch.cpp

```diff
@@ -1662,6 +1662,13 @@ bool VNetRouteOrch::doRouteTask<VNetVrfObject>(const string& vnet, IpPrefix& ipP
     }
 
     auto *vrf_obj = vnet_orch_->getTypePtr<VNetVrfObject>(vnet);
+
+    if (op == DEL_COMMAND && !vrf_obj->hasRoute(ipPrefix))
+    {
+        SWSS_LOG_WARN("VNET %s, Tunnel Route %s do not exist", vnet.c_str(), ipPrefix.to_string().c_str());
+        return true;
+    }
+
```

Comment:
Can you elaborate on how it possible to achieve a flow where the Redis DB generates a DEL event if the entry doesn't exist?

Reply:
I tested this on the machine.

Add sub route to Vnet1:
(App DB)VNET_ROUTE_TABLE:Vnet1|123.0.0.0/24

Then kill orch add del the sub route.
redis-cli 
127.0.0.1:6379> SMEMBERS VNET_ROUTE_TUNNEL_TABLE_DEL_SET
1) "Vnet1|123.0.0.0/24"
127.0.0.1:6379> SMEMBERS VNET_ROUTE_TABLE_DEL_SET
1) "Vnet1|123.0.0.0/24"